packages/core-components/src/components/Select/Select.tsx

```diff
@@ -101,12 +102,12 @@ const useStyles = makeStyles(
   { name: 'BackstageSelect' },
 );
 
-type Item = {
+export type Item = {
```

Comment:
Lets make this type name something less generic. Like `SelectItem` or something?

Reply:
Ok, will be done